lib/jsdom/living/fetch/reservedHeaders.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
```

Comment:
Filename should be `reserved-headers`.

Although, why are these called "reserved headers", instead of e.g. "header classes" or "header types" or similar?

Reply:
Naming things is hard.